V22.1 update

aaron-boranian opened the pull request:
Update to EnergyPlus v22.1.

Make soil properties used by Kiva consistent with [CSE test suite](https://github.com/bigladder/cse-resnet-ref).

aaron-boranian committed:
Update to EnergyPlus v22.1.

- Update `Version` object.
- Add new input fields, define additional input fields that are now required, and update output variable names.
- Add new MD file that tracks all object types used and what template files contain them.

aaron-boranian committed:
Make soil properties used by Kiva consistent with CSE test suite.

- Soil conductivity = 1.73 W/m-K (1.0 Btu/hr-ft-F)
- Soil specific heat = 419 J/kg-K (0.1 Btu/lb-F)
- Soil density = 1842.12 kg/m3 (115 lb/ft3)
- These are all default values for each property in both CSE and EnergyPlus.

shorowit commented:
I recommend also setting the [Foundation:Kiva initial indoor air temperature](https://bigladdersoftware.com/epx/docs/22-1/input-output-reference/group-advanced-surface-concepts.html#field-initial-indoor-air-temperature) to 68F (20C) for the four basement cases, otherwise EnergyPlus will use 78F due to [this bug](https://github.com/NREL/EnergyPlus/issues/9692). This can change the heating loads by upwards of 2+ MBtu/yr.

nealkruis commented on base.pxt:
```diff
@@ -801,7 +803,13 @@ WindowProperty:FrameAndDivider,
   0,                       !- Divider Visible Absorptance
-  0.9;                     !- Divider Thermal Hemispherical Emissivity
+  0.9,                     !- Divider Thermal Hemispherical Emissivity
+  0.0,                     !- Outside Reveal Solar Absorptance
+  0.0,                     !- Inside Sill Depth
+  0.0,                     !- Inside Sill Solar Absorptance
+  0.0,                     !- Inside Reveal Depth
+  0.0,                     !- Inside Reveal Solar Absorptance
+  CurtainWall;             !- NFRC Product Type for Assembly Calculations
```
Is this really necessary? There isn't a default?

aaron-boranian replied:
The `\min-fields` IDD rule increased from 20 => 26 for the `WindowProperty:FrameAndDivider` object, making these 6 inputs now required. All of these values are the IDD defaults, so I could reset them to blank fields.

nealkruis commented on energyplus-version-updates.md:
```diff
@@ -0,0 +1,115 @@
+# Review this file when updating EnergyPlus versions
```
What is this file?

aaron-boranian replied:
This is a file that lists all object types used in this project, along with the files that they are found in. This is helpful when diff'ing IDDs between old & new EnergyPlus versions, since when you find an object with IDD diffs you can then check this file to know if you need to update this project accordingly (and what file to update).

aaron-boranian replied:
I'll go ahead and delete this file from the repo, since it may be confusing for others.

aaron-boranian committed:
Update initial indoor air temperature for Kiva.

- Hard-set to 20C (68F) to match heating thermostat setpoint. Kiva is used for Slab (L302 & L304) and Basement (HVAC 3a-3d, L322, L324) cases in CO Sprints only, so this only affects results for these test cases.
- Align *Name* input field comment with rest of comments in Foundation:Kiva objects.

aaron-boranian commented:
@shorowit thanks for the recommendation. My latest commit hard-sets the initial indoor air temperature to 20C (68F) for Kiva in Slab and Basement cases. Kiva is applied only to the following tests (along with changes in results):

Slabs
- Std 140 Heating tests L302 & L304 (increased ~0.5 MBtu)
 
Basements
- DSE tests 3a-3d  (increased ~0.2 MBtu)
- Std 140 Heating tests L322 & L324  (increased ~2.0 MBtu)

After applying this change, I confirmed that results only updated for these tests, and have already updated the v22.1 columns in the [RESNET Acceptance Criteria XLSX files on OneDrive](https://onedrive.live.com/?id=BD3EC5E7E82198AD%21177&cid=BD3EC5E7E82198AD) accordingly.

aaron-boranian committed:
Delete energyplus-version-updates.md

- This file was for internal reference and may be confusing for others reviewing the repo.

nathancoliver committed:
Merge remote-tracking branch 'origin/master' into v22-1-update.

nathancoliver committed:
Merge remote-tracking branch 'origin/master' into v22-1-update
